source/plugins/ruby/out_mdm.rb

```diff
@@ -102,6 +102,7 @@ def start
           else
             @http_client = Net::HTTP.new(@post_request_uri.host, @post_request_uri.port)
           end
+          @http_client.instance_eval { @ssl_context = OpenSSL::SSL::SSLContext.new(:TLSv1_2) }
```

Comment:
I think, you have mentioned  ciphers being picked up randomly. is the cipher not issue?

Reply:
ciphers are picked on based on tls protocol. If you see the error, in our case client is picking ssl_v3 and server doesn;t support, and we see the error on the client side.